L1Trigger/L1TGlobal/interface/GlobalBoard.h

```diff
@@ -226,6 +231,7 @@ namespace l1t {
     BXVector<const l1t::L1Candidate*>* m_candL1Tau;
     BXVector<const l1t::L1Candidate*>* m_candL1Jet;
     BXVector<const l1t::EtSum*>* m_candL1EtSum;
+    BXVector<const l1t::EtSum*>* m_candL1EtSumZdc;
     BXVector<const GlobalExtBlk*>* m_candL1External;
```

Comment:
Since I'm looking at this... _after_ the ZDC work is done, could you come back to this file and check
  - why do these need to be pointers in the first place, instead of being regular objects (or at most `std::optional` objects) ?
  - if they need to be pointers, can you wrap them as `std::unique_ptr` ?